plugins/by-name/quicker/settings-options.nix

```diff
@@ -0,0 +1,156 @@
+lib:
+let
+  inherit (lib) types mkOption;
+  inherit (lib.nixvim) defaultNullOpts literalLua;
+in
+{
+  opts =
+    defaultNullOpts.mkAttrsOf types.anything
+      {
+        buflisted = false;
+        number = false;
+        relativenumber = false;
+        signcolumn = "auto";
+        winfixheight = true;
+        wrap = false;
+      }
+      ''
+        Local options to set for quickfix.
+      '';
+
+  use_default_opts = defaultNullOpts.mkBool true ''
+    Set to `false` to disable the default options in `opts`.
+  '';
+
+  keys =
+    defaultNullOpts.mkListOf
+      (types.submodule {
+        freeformType = with types; attrsOf anything;
+
+        options = {
+          __unkeyed__key = mkOption {
+            type = with types; maybeRaw str;
+            example = ">";
+            description = ''
+              Key sequence.
+            '';
+          };
+
+          __unkeyed__command = mkOption {
+            type = with types; maybeRaw str;
+            example = literalLua "require('quicker').collapse";
+            description = ''
+              Command to run.
+            '';
+          };
+        } // lib.nixvim.keymaps.mapConfigOptions;
+      })
+      [
+        {
+          __unkeyed__key = ">";
+          __unkeyed__command = "<cmd>lua require('quicker').toggle_expand()<CR>";
+          desc = "Expand quickfix content";
```

Comment:
Why these key names? Given we're converting them to a list-like table, I'm kinda uncomfortable relying on the nix attr-name sorting here without having numeric indexes :thinking: